miden-remote-provers/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+extern crate alloc;
+
+use alloc::string::String;
+
+use thiserror::Error;
+
+#[cfg(feature = "tx-prover")]
+pub mod generated;
+
+#[cfg(feature = "tx-prover")]
+mod prover;
+#[cfg(feature = "tx-prover")]
+pub use prover::RemoteTransactionProver;
+
+/// Contains the protobuf definitions
+pub const PROTO_MESSAGES: &str = include_str!("../../proto/api.proto");
```

Comment:
I would change the doc comment to something like: "Protobuf definitions for the Miden proving service".

Also nit: maybe we should change the constant name to something like `SERVICE_PROTO`?